tests/e2e/kueue_test.go

```diff
@@ -164,6 +172,36 @@ func (tc *KueueTestCtx) testOwnerReferences() error {
 	return nil
 }
 
+func (tc *KueueTestCtx) validateVAPReady() error {
+	// if ocp is 4.17+ then VAP and VAPB should be created
+	if cluster.GetClusterInfo().Version.GTE(semver.MustParse("4.17.0")) {
+		keyvap := types.NamespacedName{Name: "kueue-validating-admission-policy"}
+		vap := &unstructured.Unstructured{}
+		vap.SetGroupVersionKind(gvk.ValidatingAdmissionPolicy)
+
+		err := tc.testCtx.customClient.Get(tc.testCtx.ctx, keyvap, vap)
+		if err != nil {
+			return fmt.Errorf("expect validatingadminssionpolicy to be found but got error: %w", err)
+		}
+		if len(vap.GetOwnerReferences()) == 0 {
+			return errors.New("expect VAP to have ownerref set")
+		}
+
+		keyvapb := types.NamespacedName{Name: "kueue-validating-admission-policy-binding"}
+		vapb := &unstructured.Unstructured{}
+		vapb.SetGroupVersionKind(gvk.ValidatingAdmissionPolicyBinding)
+
+		err = tc.testCtx.customClient.Get(tc.testCtx.ctx, keyvapb, vapb)
```

Comment:
would be better to use gomega's `Eventually ` so that it won't fail in case the resource takes a little time to become available